--- lincity-ng-2.0/src/lincity/fileutil.cpp~	2009-01-26 05:20:41.000000000 +0900
+++ lincity-ng-2.0/src/lincity/fileutil.cpp	2017-07-13 23:48:55.355365373 +0900
@@ -498,7 +498,7 @@
 
 void malloc_failure(void)
 {
-    printf(_("Out of memory: malloc failure\n"));
+    puts(_("Out of memory: malloc failure\n"));
     exit(1);
 }
 
--- lincity-ng-2.0/src/lincity-ng/Dialog.cpp~	2009-01-26 05:20:43.000000000 +0900
+++ lincity-ng-2.0/src/lincity-ng/Dialog.cpp	2017-07-13 23:49:41.569815446 +0900
@@ -378,7 +378,7 @@
         setParagraphN( "statistic_text", line++, outf );
     }
     if (sustain_flag){
-	    snprintf (outf, maxlength, _("Economy is sustainable"));
+	    snprintf (outf, maxlength, "%s", _("Economy is sustainable"));
         setParagraphN( "statistic_text", line++, outf );
     }
     snprintf (outf, maxlength, _("Population  %d  of which  %d  are not housed.")
