--- lincity-ng-b9062bec252632ca5d26b98d71453b8762c63173/src/lincity/fileutil.cpp~	2021-11-25 16:42:03.000000000 +0000
+++ lincity-ng-b9062bec252632ca5d26b98d71453b8762c63173/src/lincity/fileutil.cpp	2022-11-23 15:07:17.930255408 +0000
@@ -469,7 +469,7 @@
 
 void malloc_failure(void)
 {
-    printf("%s",_("Out of memory: malloc failure\n"));
+    puts(_("Out of memory: malloc failure\n"));
     exit(1);
 }
 
