--- lincity-ng-1.1.1/lincity-ng.desktop~	2007-08-22 21:19:28.000000000 +0200
+++ lincity-ng-1.1.1/lincity-ng.desktop	2007-08-22 21:21:48.000000000 +0200
@@ -5,9 +5,12 @@
 Name=Lincity-NG
 GenericName=City Simulation Game
 GenericName[de]=Stadtsimulationsspiel
-Comment=Plan and Build a city and take care of transport, economics, electricityand others.
+GenericName[pl]=Symulator miasta
+Comment=Plan and Build a city and take care of transport, economics, electricity and others
+Comment[pl]=Zaplanuj i zbuduj miasto oraz dbaj o transport, ekonomie, elektryczność i inne rzeczy
 Icon=lincity-ng.png
 Exec=lincity-ng
 Terminal=false
 StartupNotify=false
 Categories=Game;Simulation;
+# vi: encoding=utf-8
